committee-code-of-conduct/bootstrapping-process.md

```diff
@@ -0,0 +1,36 @@
+# Bootstrapping Document for Operator Framework Code of Conduct Committee
+
+This document (created by the Operator Framework community chairs) outlines what the Code of Conduct Committee (CoCC) is responsible for, how to create it, and the rough processes for them to get started.
+
+## Objectives of the CoCC
+* Maintain the Code of Conduct (CoC) document and iterate as needed.
+  * All CoC revisions must be approved by the community chairs.
+  * Currently, we use the CNCF CoC. Any addendums or changes based off learnings in the Operator Framework community should be owned by this body of people, the Operator Framework Code of Conduct Committee (CoCC).
+* Determine and make transparent how CoC issues and incidents are reported and handled in the community.
+* Enforce the Code of Conduct within the Operator Framework community.
+  * Discuss with parties affected
+  * Come up with resolution
+  * For moderation on a day-to-day basis, work with the existing community-management subproject under SIG Contributor Experience
+* Figure out how to keep and then keep a record of CoC reports and outcomes for precedent _This is something we are still actively discussing on how to do and is not fully hardened._
+
+## Formation of the Code of Conduct Committee (CoCC):
+* The CoCC consists of 5 members. In the first election, the top 3 voted people will be appointed 2 year terms and the other 2 members will be appointed for a 1 year term.
```

Comment:
Two years seems like a long time.   Perhaps everyone gets a 1 year term but every year only half of the folks are up for re-election.  So after you get the initial 5 members, 3 of the folks up for election next year and then the other two up for election the year after that.  That way you always have the opportunity for new folks on the committee but you also have continuity since not everyone is up for election at the same time.

Reply:
Your wording is confusing.

>not everyone is up for election at the same time.

You can't stagger election years if everyone has a 1 year term. Are you suggesting that everyone has a 1 year term, then at the end of that year 3 members can be re-elected but 2 people cannot until the following year?